packages/source-iottwinmaker/src/index.ts

```diff
@@ -25,3 +25,4 @@ export {
 } from './utils/dataBindingTemplateUtils';
 export * from './data-binding-provider/types';
 export type { FetchEntityErrorMeta } from './metadata-module/types';
+export { SceneMetadataModule } from './scene-module/SceneMetadataModule';
```

Comment:
you shouldn't need to export it since it's the concrete class. you should use `TwinMakerSceneMetadataModule` when referencing as type

Reply:
Updated!